.github/workflows/build.yml

```diff
@@ -4,25 +4,23 @@ name: Build
 on:
   push:
   pull_request:
-  schedule:
-    - cron: '0 0 * * 0'
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Build static website
-        run: docker run --rm -v $PWD:/srv/jekyll -eJEKYLL_UID=$UID jekyll/builder:pages jekyll build --config _config.yml,_prod.yml
-      - name: Run htmlproofer
-        run: docker run --rm -v $PWD/_site:/site/about jekyll/builder:pages /usr/gem/bin/htmlproofer /site --url-ignore "/jupyter,/webclient/,/cell/,/tissue/,/,"
+        run: ./build.sh
+        env:
+          UID: 1001
       - name: Create website archive
-        run: tar -zcvf idr.openmicroscopy.org.tar.gz -C _site ./
+        run: tar -zcvf ${{ github.event.repository.name }}.tar.gz -C _site ./
```

Comment:
This is so that the workflow can be a template?

Reply:
Yes that's the idea. https://github.com/ome/www.openmicroscopy.org is the next one where I'd like to test this

Reply:
Cool. Then I guess if you wanted to keep the cronjob you could introduce a variable.

Reply:
Agreed. The alternative is to use consume `github.repository` (https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions) and do some string manipulation to extract the name